lib/bango/tests/test_forms.py

```diff
@@ -90,7 +90,7 @@ class TestPackage(APITest):
     def test_no_auth(self, clean):
         form = PackageForm(good_address)
         ok_(form.is_valid())
-        ok_(not 'eventNotificationURL' in form.bango_data)
+        ok_('eventNotificationURL' not in form.bango_data)
```

Comment:
Both work, but the latter is preferred. I fixed a few of these in zamboni and the tests all passed.